cmd/celestia-appd/cmd/root.go

```diff
@@ -135,11 +133,12 @@ func initRootCmd(rootCmd *cobra.Command, encodingConfig encoding.Config) {
 		genutilcli.GenTxCmd(app.ModuleBasics, encodingConfig.TxConfig, banktypes.GenesisBalancesIterator{}, app.DefaultNodeHome),
 		genutilcli.ValidateGenesisCmd(app.ModuleBasics),
 		tmcli.NewCompletionCmd(rootCmd, true),
-		debugCmd,
+		debug.Cmd(),
 		config.Cmd(),
 		commands.CompactGoLevelDBCmd,
 		addrbookCommand(),
 		downloadGenesisCommand(),
+		addrConversionCmd(),
```

Comment:
@cmwaters - You originally suggested moving this under the debug subcommand. This makes sense, but would require adding this content via a PR to [celestiaorg/cosmos-sdk](https://github.com/celestiaorg/cosmos-sdk) instead, then subsequently referencing the command from here.

Reply:
I think it's okay for now. `celestia-appd` is a messy place and hasn't had much intentional design around it. I'm punting on this mainly because I'd prefer to see a single binary / CLI in the future and there we can worry about how things are logically set up